challenges/classes.js

```diff
@@ -1,7 +1,68 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
-// Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+/* ===== Prototype Practice ===== */
+
+// Task: You are to build a cuboid maker that can return values for a cuboid's volume or surface area. Cuboids are similar to cubes but do not have even sides. Follow the steps in order to accomplish this challenge.
+
+/* == Step 1: Base Constructor ==
+  Create a constructor function named CuboidMaker that accepts properties for length, width, and height
+*/
+/* == Step 2: Volume Method ==
+Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
+
+Formula for cuboid volume: length * width * height
+*/
+
+
+/* == Step 3: Surface Area Method ==
+Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
 
-// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
\ No newline at end of file
+Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
+*/
+
+class CuboidMaker {
+    constructor(length, width, height) {
+        this.length = length;
+        this.width = width;
+        this.height = height;
```

Comment:
What is the difference between what you have and
`constructor(parameter) {
this.length = parameter.length;
this.width = parameter.width`?

Reply:
The way I wrote it, I passed in each parameter on its own instead of passing in an object containing key value pairs of the attributes. I did it that way because I felt that the constructor was simple enough (just taking three integers/floats) that the numbers could be passed in individually instead of within another object that contained properties (like we did for the lambda classes assignment). But perhaps what you wrote is more accurate to the intent of the question, if we were meant to do this exactly the same way as we did for the lambda classes assignment. 

Thanks for pointing it out!